### PR TITLE
Expand REE Files documentation

### PR DESCRIPTION
**Pull request by francescpuig7**
### Description:

Add documentation for: AGCL, AGCLACUM, AGCLACUM5, MEDIDAS CNMC and validations REE files

### Tasks:

- [x] Docs
- [x] Translations
- [x] Images

### Links:

* [Distr/perfils#validacions de mesures](http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/ca/distri/perfiles/#validacions-de-mesures)
* [Distr/perfiles#validaciones de medidas](http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/es/distri/perfiles/#validaciones-de-medidas)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/perfiles.md [hunk 1]
```diff
@@ -363,6 +413,20 @@ A l'apartat de REE es generen els següents fitxers:
...
 ![](_static/perfiles/menu_ree.png)
 
+## Menú de generació de fitxers de la CNMC
+
+Des de l'apartat de l'antic projecte SICILIA, es pot generar el fitxer MEDIDAS.
+Per gernerar-lo el trobem a la següent ruta **Adminitració Pública  > CNMC >
+SICILIA > Generar MEDIDAS`.**
+
+!!! Info "Nota"
+  A diferència del fitxer MEDIDAS per REE, aquest es generen n fitxers per n
```
Comment (jaumef) [inline docs/distri/perfiles.md]: Ha d'haver 4 espais (identació), sino no ho detecta amb el widget i el text desapareix

Comment (jaumef) [inline docs/distri/perfiles.md]: (ES) diferéncia → diferencia

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/perfiles.md (hunk 1 above)
Comment (francescpuig7) [reply docs/distri/perfiles.md]: Done via: https://github.com/gisce/powerp-docs/pull/294/commits/dc5064bedac22eea603b31590dcfc5ef1b4e8d1c

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_expand_re_doc/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by jaumef** [comment]
Test hook is not working properly... https://travis-ci.org/gisce/powerp-docs/builds/365184307
PASSED

**Pull request merged**